methods.py

```diff
@@ -62,4 +60,3 @@ def exportAsPickle(x,y):
     return "Saved as: " + z
 
 

```

Comment:
General question: Do you already have a list of the regex commands you would like to use for Orbitrap?

Reply:
A next step would be combining these modular functions, to select all spectra belonging to orbitrap, where the search criteria are predefined for Orbitrap. It might also be a good option to do this in a jupiter notebook, so you can right away check if it was successful.

Reply:
> 

Yes! In the main.py file there's a set of statements that relate to every instrument type with regexp (including metacharacters)